example3/OpenTelemetry.Metric.Api/Meter.cs

```diff
@@ -0,0 +1,173 @@
+using System.Collections.Generic;
+using Microsoft.Diagnostics.Metric;
+
+namespace OpenTelemetry.Metric.Api
+{
+    public interface IMeter
+    {
+        Counter CreateCounter(string name);
+        Counter CreateCounter(string name, Dictionary<string,string> labels);
+        Counter CreateCounter(string name, Dictionary<string,string> labels, params string[] dimNames);
+
+        Counter1D CreateCounter(string name, string dn1);

```

Comment:
From a initial glance at the PR, this looks like a user needs to take a dependency on Otel.Api to start emitting metric? Or can they do it with pure .NET (i.e by depending on Ms.Diagnostic.Metric alone? which I would assume will become System.Diag.DiagnosticSource.)

Reply:
My current stance is that I am aiming to have the vast majority of scenarios (95+%) are doable with .NET API alone. I'm not sure yet what @victlu had in mind with these Counter#D types : )

Reply:
The Counter#D types was to limit # of dim values taken in Add() calls.  Otherwise, it's an open string array with possible mismatch in # of values provided.